app/src/js/components/Providers/add.js

```diff
@@ -9,23 +9,55 @@ import { isValidProvider } from '../../utils/validate';
 
 const SCHEMA_KEY = 'provider';
 
-const AddProvider = ({ providers }) => (
-  <div className="add_providers">
-    <Helmet>
-      <title> Add Provider </title>
-    </Helmet>
-    <AddRecord
-      baseRoute="/providers/provider"
-      createRecord={createProvider}
-      primaryProperty="id"
-      schemaKey={SCHEMA_KEY}
-      state={providers}
-      title="Create a provider"
-      validate={isValidProvider}
-      validationError={'Concurrent Connection Limit cannot be a negative value'}
-    />
-  </div>
-);
+const nonSftpExcludes = ['privateKey', 'cmKeyId'];
+const nonHttpsExcludes = ['allowedRedirects', 'certificateUri'];
+const s3Excludes = [
+  ...nonSftpExcludes,
+  ...nonHttpsExcludes,
+  'username',
+  'password',
+  'port',
+];
```

Comment:
Can you make a quick ticket to document these and then do that ticket too? 😬

Reply:
After digging through the core docs, I realized the info for providers was moved to the data management types doc. I updated that here: https://github.com/nasa/cumulus/pull/2604
I also opened a quick PR to add some missing fields to the api docs, which is here: https://github.com/nasa/cumulus-api/pull/316

Reply:
Thanks for getting those.